src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <div className="cont-button">
+      <div className="btn-row">
+        <Button name="Ac" color="rgb(224, 223, 219)" />
+        <Button name="+/-" color="rgb(224, 223, 219)" />
+        <Button name="%" color="rgb(224, 223, 219)" />
+        <Button name="÷" color="rgb(224, 223, 219)" />
```

Comment:
Good job making the necessary corrections seems you did more 😄 
You don't need to add color to the signs on the right side(signs like: `÷`, `x`,  `-`, `+`, `=`), we want them to have default color which is `orange` right.
kindly remove the color 👍 

Example: `<Button name="÷" />`


After I removed `rgb` color on side signs;
![image](https://user-images.githubusercontent.com/31897434/81672968-d3c6e080-9442-11ea-8c7a-e50d9850ca71.png)